packaging/docker-compose.yml

```diff
@@ -1,20 +1,19 @@
-version: '3.4'
-
 services:
   packager:
     build:
       context: .
       args:
-        - AWSCLI_VERSION=${AWSCLI_VERSION:-2.1.36}
-        - S3CMD_VERSION=${S3CMD_VERSION:-2.1.0}
+        - AWSCLI_VERSION=${AWSCLI_VERSION:-2.26.5}
+        - S3CMD_VERSION=${S3CMD_VERSION:-2.4.0}
```

Comment:
Can't we use relaxed format for versioning? It would be good to use at least the latest patched version.

Reply:
Wouldn't using the exact versioning be better for pinning down possible issues and leaving us in control?

Reply:
I agree.